scipy/interpolate/_rbfinterp.py

```diff
@@ -396,7 +396,7 @@ def _chunk_evaluator(
             shift,
             scale,
             coeffs,
-            memory_budget=None
+            memory_budget=1000000
```

Comment:
I'd typically go for `if memory_budget is None: memory_budget=int(1e6)` but this is really minor and is not worth the churn unless you want to fix in a sequel (I'm fine either way)